client/src/components/Ticketing/event/TicketPicker.tsx

```diff
@@ -462,7 +462,7 @@ const TicketPicker = (props: TicketPickerProps): ReactElement => {
           qty > selectedTicket.availableseats ||
           (selectedTicketType.name === 'Pay What You Can' && (payWhatPrice == null || payWhatPrice < 0))
         }
-        className='disabled:opacity-30 disabled:cursor-not-allowed py-2 px-3 mt-7 bg-blue-500 text-white enabled:hover:bg-blue-600 rounded-xl'
+        className='disabled:opacity-30 disabled:cursor-not-allowed py-2 px-3 mt-7 bg-blue-500 text-white enabled:hover:bg-blue-600 rounded-2xl'
```

Comment:
Any particular reason why we're changing the rounding? It looks like most other buttons use `rounded-xl`

Reply:
No, that was a mistake. Removed.